cpp/ql/test/library-tests/vector_sizes/code.c

```diff
@@ -0,0 +1,6 @@
+// semmle-extractor-options: --clang --edg --trap_container=folder --edg --trap-compression=none
```

Comment:
I don't think we want the `--trap*` flags in the test. Note that you can `export SEMMLE_TRAP_CONTAINER=folder` to override it on the commandline.

Reply:
Pushed new changed which drop the --trap options